### DISCUSSION TITLE
[功能建议] 请问是否能为 自定义顶栏-收藏 添加一个删除选项

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3652] [功能建议 / Ideas]
hmmm...这会是一个对**主页被屏蔽后的用户**友好的选项，这种类型的用户是无法打开`https://space.bilibili.com/*/favlist`这种收藏页面的

如果可以做成一个能够批量删除的选项就更棒了！

感谢作者大大！辛苦了！

**Comment by Geeyun-JY3** [comment]
重复：#916

**Comment by Tinhone** [comment]
阿巴阿巴，抱歉
<del>所以是两年了还没做出来吗</del>

**Reply by Geeyun-JY3** [reply]
~~不知道~~

拿出来晒晒也不是坏事

**Comment by the1812** [comment]
**主页被屏蔽后的用户**是啥?

**Reply by Tinhone** [reply]
一些主页`space.bilibili.com`会显示
```
错误码：-404
fetch info 啥都木有
```
且**未注销账号**的用户

这类用户虽仍然可以正常使用bilibili的大部分功能（封号除外），但诸如`space.bilibili.com/[uid]` `space.bilibili.com/[uid]/favlist`等链接均无法被本人或他人访问
